style.css

```diff
@@ -1,3 +1,98 @@
-h1 {
-  color: aquamarine;
+:root {
+  --purple: #6070ff;
+  --mainFont: 'Poppins';
+}
+
+html {
+  box-sizing: border-box;
+  font-family: var(--mainFont);
+  font-size: 62.5%;
+}
+
+html,
+body {
+  padding: 0;
+  margin: 0;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 85%;
+  margin: 0 auto;
+}
+
+header p,
+nav {
+  font-size: 2rem;
+  color: var(--purple);
+}
+
+header p {
+  font-weight: 700;
+}
+
+/* Hero */
+
+.hero {
+  background-image: url('./img/hero.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 63.5rem;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.hero h1 {
+  font-size: 48px;
+  font-weight: 700;
+  margin-bottom: 0.5px;
+}
+
+.hero-text p {
+  font-size: 20px;
+  line-height: 30px;
+  font-weight: 300;
+  margin-top: 5px;
+  color: #344563;
+}
```

Comment:
- [Optional] Also here, You used a px for a font. This works fine for now. But for better responsiveness, I suggest you use a responsive sizing like `rem` instead of `20px` I suggest you use something like `2rem`